src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Telemetry/VsTelemetryService.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Telemetry;
+
```

Comment:
Given this has nothing to do with the project system itself, can we put this in Microsoft.VisualStudio.Telemetry namespace?